internal/fs/inode/file_test.go

```diff
@@ -682,3 +682,39 @@ func (t *FileTest) SetMtime_SourceObjectMetaGenerationChanged() {
 	ExpectEq(newObj.Generation, o.Generation)
 	ExpectEq(newObj.MetaGeneration, o.MetaGeneration)
 }
+
+func (t *FileTest) ContentEncodingGzip() {
+	// Set up an explicit content-encoding on the backing object and re-create the inode.
+	const contentEncoding = "gzip"
+	if t.backingObj.Metadata == nil {
```

Comment:
why is this condition required for setting contentEncoding?

Reply:
This check is a copy/paste typo. Will remove.

Reply:
Done.